modules/ROOT/pages/6.0-release-notes-formatting.adoc

```diff
@@ -0,0 +1,26 @@
+= TinyMCE 6.0 Formatting changes
+:navtitle: TinyMCE 6.0 Formatting changes
+:description: TinyMCE 6.0 Formatting changes
+:keywords: releasenotes, apis
+
+This section documents the single change made to {productname} 6.0 Formatting.
+
+* xref:formatting[Formatting]
+* xref:changed-formatting[Changed Formatting]
+
+// tag::formatting[]
+[[formatting]]
+== Formatting
+
```

Comment:
Considering there's only one entry on this page, having these lines feels extraneous to me? Unsure if we might be using them elsewhere to pull this into a larger page or something though